src/Microsoft.VisualStudio.Threading.Analyzers/VSSDK008UseAwaitInAsyncMethodsCodeFix.cs

```diff
@@ -135,7 +135,7 @@ public override string Title
                 if (this.AlternativeAsyncMethod != string.Empty)
                 {
                     // Replace the member being called and await the invocation expression.
-                    var asyncMethodName = SyntaxFactory.IdentifierName(this.diagnostic.Properties[AsyncMethodKeyName]);
+                    var asyncMethodName = syncMethodName.WithIdentifier(SyntaxFactory.Identifier(this.diagnostic.Properties[AsyncMethodKeyName]));
```

Comment:
No. This line changed because one of the tests I add in this PR is for generic method names, and the "before" line here was erasing the generic type argument from the method invocation. The new line preserves that.